PersonStageThree/src/com/beyond/person/core/exception/InvalidAgeException.java

```diff
@@ -0,0 +1,7 @@
+package com.beyond.person.core.exception;
+
+public class InvalidAgeException extends RuntimeException {
+    public InvalidAgeException() {
+        super("Invalid age!");
```

Comment:
@maria-salmasian Provide unique msg in exception throw place, is much better then hardcoded it on constructor

Reply:
done